quinn/src/builders.rs

```diff
@@ -222,6 +222,12 @@ impl ClientConfigBuilder {
         self
     }
 
+    /// Enable 0-RTT, by configuring TLS to accept early data.
```

Comment:
```suggestion
    /// Enable 0-RTT
```
TLS's own terminologoy is a distraction here.